### PR TITLE
94: Fix accordions to collapse others.

### PR DESCRIPTION
**Pull request by marklise**

### CODE REVIEW

File: src/app/enter-data/accordion-manager/accordion-manager.component.ts [hunk 1]
```diff
@@ -37,6 +37,43 @@ export class AccordionManagerComponent implements OnDestroy {
...
+    let mappedAccordionItems = {
+      'collapsefrontcountryCamping': 'frontcountryCamping',
+      'collapsefrontcountryCabins': 'frontcountryCabins',
+      'collapsegroupCamping': 'groupCamping',
+      'collapsedayuse': 'dayuse',
+      'collapsebackcountryCamping': 'backcountryCamping',
+      'collapsebackcountryCabins':'backcountryCabins',
+      'collapseboating': 'boating',
+    };
+    console.log('Current Accordion Clicked:', currentId);
```
Comment (cameronpettit) [inline src/app/enter-data/accordion-manager/accordion-manager.component.ts:50]: console.log(). Theres a few more.

File: src/app/enter-data/accordion-manager/accordion-manager.component.ts [hunk 2]
```diff
@@ -37,6 +37,43 @@ export class AccordionManagerComponent implements OnDestroy {
...
+            // Check if it's open.
+            if (item && item.classList.contains('show')) {
+              let hideItem = document.getElementById(mappedAccordionItems[keys[key]])?.firstChild;
+              if (hideItem) {
+                hideItem.dispatchEvent(clickEvent);
+              }
+            }
+        } else {
+          // console.log(`Not touching ${keys[key]}`);
+        }
```
Comment (cameronpettit) [inline src/app/enter-data/accordion-manager/accordion-manager.component.ts:72]: `else` unnecessary

**Review by cameronpettit — CHANGES_REQUESTED** [review]

File: src/app/enter-data/accordion-manager/accordion-manager.component.ts (hunk 2 above)
Comment (marklise) [reply src/app/enter-data/accordion-manager/accordion-manager.component.ts]: Fixed in b510bf1

File: src/app/enter-data/accordion-manager/accordion-manager.component.ts (hunk 1 above)
Comment (marklise) [reply src/app/enter-data/accordion-manager/accordion-manager.component.ts]: Fixed in b510bf1

**Review by cameronpettit — APPROVED** [review]

**Review by danieltruong — APPROVED** [review]

**Pull request merged**